src/control/group/mod.rs

```diff
@@ -24,6 +24,13 @@ cfg_if! {
     ))] {
         mod neon;
         use neon as imp;
+    } else if #[cfg(all(
+        target_arch = "loongarch64",
+        target_feature = "lsx",
+        not(miri),
```

Comment:
This should be under the "nightly" feature until loongarch intrinsics are stabilized.

Reply:
I was also going to mention this-- that way loongarch still works on stable.

Reply:
Thanks. It's done.